template/src/components/treeList/TreeItem.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-list-item>
+    <v-list-item-title
+      ><router-link
```

Comment:
Strange formatting of html tags closures here, quite hard to read. Is that also prettier's job or maybe editor's settings? 🤔